crates/cloudagent-python/src/web.rs

```diff
@@ -55,29 +57,30 @@ impl CloudAgent {
             None => client,
         };
 
-        trace!("About to send request:\n{:#?}", client);
+        log_trace!("About to send request:");
+        log_trace!("{:#?}", client);
         match client.send().await {
             Ok(res) => {
                 let status_code = res.status().as_u16();
-                trace!("Got {} response:\n{:#?}", status_code, res);
+                log_info!("Got {} response:", status_code);
+                log_info!("{:#?}", res);
```

Comment:
I think this is pretty verbose for "info" level output.

```suggestion
                log_trace!("Got {} response:", status_code);
                log_trace!("{:#?}", res);
```